src/osd/osd_types.h

```diff
@@ -4631,6 +4710,7 @@ inline std::ostream& operator<<(std::ostream& out, const pg_log_t& log)
  */
 struct pg_missing_item {
   eversion_t need, have;
+
```

Comment:
nit: do we need this blank?

Reply:
Done, please continue review https://github.com/ceph/ceph/pull/52871